sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -400,6 +479,11 @@ def from_predictions(
 
         name = name if name is not None else "Classifier"
 
+        if plot_chance_level:
+            pos_prevalence = Counter(y_true)[pos_label] / len(y_true)
+        else:
+            pos_prevalence = 0
```

Comment:
If my previous comment is taken into account, this line should be modified not to raise the warning/error.